modules/vpc/outputs.tf

```diff
@@ -3,6 +3,11 @@ output "vpc_id" {
   value       = "${aws_vpc.vpc.id}"
 }
 
+output "vpc_cidr" {
```

Comment:
Please name this `vpc_cidr_block` to keep the `cidr_block` part of the name consistent with `aws_vpc.vpc.cidr_block`.

Reply:
will do :+1: